landlab/components/river_bed_dynamics/river_bed_dynamics.py

```diff
@@ -0,0 +1,3433 @@
+            A[self._grid.horizontal_links] = (
+                h_links[self._grid.horizontal_links] * self._grid.dx
+            )
+            A[self._grid.vertical_links] = (
+                h_links[self._grid.vertical_links] * self._grid.dy
+            )
+
+            P = np.zeros_like(h_links)
+            P[self._grid.horizontal_links] = (
+                self._grid.dx + 2 * h_links[self._grid.horizontal_links]
+            )
+            P[self._grid.vertical_links] = (
+                self._grid.dy + 2 * h_links[self._grid.vertical_links]
+            )
+            Rh = (
+                A / P
+            )  # Rh = wetted area / wetted perimeter#Rh = wetted area / wetted perimeter
+            self._shear_stress = self._rho * self._g * Rh * Sf
+        else:
+            # Equation is τ = Rho * g * h * Sf """
+            self._shear_stress = self._rho * self._g * h_links * Sf
+
+        # Apply a boundary condition of zero flux at link next to borders
+        self._shear_stress[self._links_at_border_cells] = 0
+
+        # Direction of flux will be recovered at the end of the bedload
+        # transport routine
+        self._shear_stress_total = np.abs(self._shear_stress)
+
+        # Now we write the shear stress field into the grid so other components
+        # or postprocess functions can access it
+        self._surface_water__shear_stress_link = self._shear_stress_total
+
+    def bedload_equation_Parker_1990(self):
+        """Surface-based bedload transport equation of Parker 1990
+
+        G. Parker (1990) Surface-based bedload transport relation for gravel rivers,
+        Journal of Hydraulic Research, 28:4, 417-436, DOI: 10.1080/00221689009499058
+        """
+
+        # Variables definition - All these variables are updated each time step.
+        # Therefore, we read them again to ensure they are updated
+
+        grain_size_frequency = self._bed_surface__grain_size_distribution_link
+        grain_size_geometric_mean = self._bed_surface__geometric_mean_size_link
+        grain_size_geometric_standard_deviation = (
+            self._bed_surface__geometric_standard_deviation_size_link
+        )
+
+        shear_stress_star_sg = self._shear_stress_total / (
+            self._rho * self._R * self._g * (grain_size_geometric_mean / 1000)
+        )
+        self._phi_sgo = shear_stress_star_sg / self._shear_stress_star_rsgo
+
+        self.strainFunctions()
+        self._omega = 1 + (
+            np.log2(grain_size_geometric_standard_deviation) / self._sigma0
+        ) * (self._omega0 - 1)
+
+        Di_Dsg = np.tile(self._grain_size, (self._grid.number_of_links, 1)) / (
+            np.reshape(grain_size_geometric_mean, [self._grid.number_of_links, 1])
+        )
+        phi_sgo = np.reshape(self._phi_sgo, [self._phi_sgo.shape[0], 1])
+        omega = np.reshape(self._omega, [self._omega.shape[0], 1])
+        phi_i = omega * phi_sgo * (Di_Dsg) ** -self._beta
+
+        G = np.zeros_like(phi_i)
```

Comment:
It would be better to use a more descriptive variable name (and, use *snake_case* for variables).

Reply:
Thanks, done! 
Some variables are called based on their names in the original publications, but most of their names were slightly modified to be more descriptive